Fresh commit of portinstall: installing software using FreeBSD's ports.

berenddeboer opened the pull request:
Hi Michael, I dropped my repository and forked again. Simply couldn't get it working otherwise.

Here a fresh pull request for installing software using FreeBSD ports.

stoned committed:
Add Jinja2 filter |bool : return boolean interpretation of the value

'yes', 'on', '1', 'true', insensitively, and 1 are true,
 everything else is false

berenddeboer committed:
Fresh commit of portinstall: installing software using FreeBSD's ports.

arsatiki committed:
Fix service enable on FreeBSD

Some services have a knob (i.e. rc.conf setting) whose name
differs from that of the script. For example, lockd process
is controlled with a script called lockd, but the rc.conf
value is rpc_lockd_enable.

Fixes issue #3382.

Commit:
Make the CPU facts Hyperthreading aware

Commit:
Makes host key checking the default behavior but can be disabled in ansible.cfg or by environment variable.

Commit:
Example config file should use correct config setting.

mspaulding06 committed:
Fix git submodule update when version specified

Commit:
Enable paramiko to ask whether to add keys to known hosts.

Commit:
Lock around SSH connectivity to new hosts in host checking mode such that prompts for host approval
messages do not get interlaced.

ajsalminen committed:
Use .encode() instead of str() to support unicode arguments.

str() throws an UnicodeEncodeError for code points that cannot be
represented in 7-bit ASCII. This makes it impossible to use any
non-ASCII characters in module arguments. Using encode('utf-8')
gives the desired result.

Commit:
Default to 'smart' transport, which will use OpenSSH if it can support ControlPersist.

skvidal committed:
- optimize for the extremely common case of people specifying pkg names which are, ultimately, already installed
  on a system here time went from  16s for a series of pkgs to 3s to just
  check that they were installed.

berenddeboer commented:
Just did a:

  git pull --rebase upstream devel

  git rebase

  git push

Not sure if this commit looks still good. It says files changed 11??? My apologies if I screwed this up again. It is not really clear to me why this is happening. Clearly git rebase works differently than I thought.

bcoca commented:
Sometimes its not git but github (or I get the same evil gerbils screwing
with my commits too!).

## 

Brian Coca
Stultorum infinitus est numerus
0110000101110010011001010110111000100111011101000010000001111001011011110111010100100000011100110110110101100001011100100111010000100001
Pedo mellon a minno

Commit:
Improve interlaced output prevention when asking for host key approval.

Commit:
Talk more about transports in documentation.

Commit:
Mention envrionment variable configuration.

Commit:
Document 'smart' option and connection type default changes.

Commit:
Update the releases list.

Conflicts:

	RELEASES.txt

Commit:
Update packaging changelogs.

Conflicts:

	packaging/rpm/ansible.spec

Commit:
Merge in release notes from 1.2.1

mpdehaan committed:
Closing stdin too early.

mpdehaan committed:
Close the named temporary file.

Commit:
Revert "ini_file: add support for lists of options/values"

Breaks "," values in INI files

This reverts commit af051d6f77db306a555f71121f14c745a4ed0363.

tonk committed:
Repair issue #3447

Commit:
Make it possible to tell paramiko to not record new host keys, which can be slow with a large number of hosts.
-c ssh is preferred in most cases if you have ControlPersist available, otherwise if you are comfortable you
can turn off recording while leaving host key checking on, etc.

Commit:
Update changelogs

Commit:
Update releases

Commit:
Update changelog

Commit:
Slight revisement to configuration file comment.

Commit:
Fixup changelog dates

Commit:
Fixed indention of vv() output

bcoca committed:
fixed check mode
Signed-off-by: Brian Coca <briancoca+dev@gmail.com>

mpdehaan commented:
I have to close this one due to the gerbils :(
